README.md

```diff
@@ -86,6 +86,24 @@ After the client is created, you can start sending Service Checks to Datadog. Se
 
 ## Performance / Metric drops
 
+### Monitoring this client
+
+This client automatically injects telemetry about itself in the dogstatsd stream.
+The following metrics are available:
+
+- `datadog.dogstatsd.client.packets_sent` (count): total number of datagrams successfully sent
```

Comment:
Could we make a table for this list ? 

| Metric name | Type | Description | 
| --- | --- | --- |

Reply:
Done!